Cineme.h

```diff
@@ -0,0 +1,116 @@
+
+#include "Movie.h"
+
+
+class Cinema
+{
+private:
+    vector<Movie> movies;
+    string name;
+    
+    int seats;
+
+public:
+    void printCinema()
+    {
+        cout << "Cinema: " << name << ";" << endl
+             << endl;
+    }
+    Cinema(string tmpName, int tmpSeats)
+    {
+        name = tmpName;
+
+        seats = tmpSeats;
+    }
+
+    void addNewMovieToCinema(Movie movie)
+    {
+        movies.push_back(movie);
+    }
+
+    void filterMovieByGenre(type genreFilter)
```

Comment:
The name of method shoud be in uppercase.

Reply:
Can be as it is.